tests/test_cli.py

```diff
@@ -115,6 +121,7 @@ def test_find_standard_component_name():
             ('reana', 'reana'),
             ('r-server', 'reana-server'),
             ('r-j-controller', 'reana-job-controller'),
+            ('reana-ui', 'reana-ui')
```

Comment:
Shouldn't it be `'r-ui'`?

Reply:
It has to be precisely `reana-ui` to test that `find_standard_component_name` is an idempotent operation.